pkgs/top-level/python-packages.nix

```diff
@@ -19123,6 +19123,29 @@ let
     };
   };
 
+  cmdtest = buildPythonPackage rec {
+    name = "cmdtest-${version}";
+    version = "0.15";
+    disabled = isPy3k || isPyPy;
+
+    buildInputs = with self; [ cliapp ttystatus ];
+    propagatedBuildInputs = with self; [ cliapp ttystatus markdown ];
+
+    # TODO: cmdtest tests must be run before the buildPhase
+    doCheck = false;
+
+    src = pkgs.fetchgit {
+        url = "git://git.liw.fi/cmdtest";
+        rev = "9cc49a98bd3ba164ed724b2574e40b253e8ba241";
+        sha256 = "4d898df468e6d6594e30899e6b1ddf57a572c2a2da900f973ddea17c97a51cfb";
+    };
+
+    meta = {
+      homepage = http://liw.fi/cmdtest/;
+      description = "cmdtest black box tests Unix command line tools";
```

Comment:
`description` should not contain the package name. See the nixpkgs manual for details.